flink-runtime/src/main/java/org/apache/flink/runtime/minicluster/MiniClusterConfiguration.java

```diff
@@ -176,6 +184,11 @@ public Builder setCommonBindAddress(String commonBindAddress) {
 			return this;
 		}
 
+		public Builder enableEmbeddedHaLeadershipControl(boolean enableEmbeddedHaLeadershipControl) {
```

Comment:
maybe rename into `withHaLeadershipControl` and add a description that this overrides the HA config option.